tests/percy/iframepagenavigator.js

```diff
@@ -9,24 +9,50 @@ class IframePageNavigator extends PageNavigator {
    * @param {import('puppeteer').Page} page A Pupeteer Page
    * @param {string} siteUrl A url to the index of the site
    * @param {string} iframePage The name of the iframe page
+   * @param {string} defaultLocale default locale config of the site
    */
-  constructor(page, siteUrl, iframePage) {
+  constructor(page, siteUrl, iframePage, defaultLocale='en') {
     super();
     this._page = page;
     this._siteUrl = siteUrl;
     this._iframePage = iframePage;
+    this._defaultLocale = defaultLocale;
+
+    /**
+     * Locale param for the site's url on universal page
+     * 
+     * @type {string}
+     */
+    this._localeUniversalUrlPath = '';
+
+    /**
+     * Locale param for the site's url on vertical page
+     * 
+     * @type {string}
+     */
+    this._localeVerticalUrlPath = '&verticalUrl=';
+  }
+  
+  /**
+   * Sets locale param for the site's url based on given locale
+   * 
+   * @param {string} locale 
+   */
+  setCurrentLocale(locale) {
+    this._localeUniversalUrlPath = locale === this._defaultLocale? '' : '&verticalUrl=' + locale;
```

Comment:
nit: we usually put a space before the question mark when using the ternary operator